draft-ietf-dots-requirements.md

```diff
@@ -196,18 +196,6 @@ Heartbeat:
 : A message transmitted between DOTS agents over the signal channel, used as a
   keep-alive and to measure peer health.
 
-Client signal:
```

Comment:
The term `client signal` is used in the architecture draft, but it doesn't seem to be in any of the other drafts adopted by the WG. It seems like the requirements covering the ability to request mitigation and to receive feedback cover these definitions, so I'm comfortable removing them.